reference/features/sorting.md

```diff
@@ -0,0 +1,200 @@
+:::
+
+### Adding attributes to `sortableAttributes`
+
+After you have decided which fields you will allow your users to sort on, you must add their attributes to the [`sortableAttributes` index setting](/reference/api/sortable_attributes.md).
+
+`sortableAttributes` accepts an array of strings, each corresponding to one attribute. Note that the attribute order in `sortableAttributes` has no impact on sorting.
+
+#### Example
+
+Suppose you have collection of books containing the following fields:
+
+```json
+[
+  {
+    "id": 1,
+    "title": "Solaris",
+    "author": "Stanislaw Lem",
+    "genres": ["science fiction"],
+    "price": 5.00
+  },
+  {
+    "id": 2,
+    "title": "The Parable of the Sower",
+    "author": "Octavia E. Butler",
+    "genres": ["science fiction"],
+    "price": 10.00
+  },
+  {
+    "id": 3,
+    "title": "The Dispossessed",
+    "author": "Ursula K. Le Guin",
+    "genres": ["science fiction", "anarchism"],
+    "price": 3.00
+  },
+  {
+    "id": 4,
+    "title": "Gender Trouble",
+    "author": "Judith Butler",
+    "genres": ["feminism", "philosophy"],
+    "price": 10.00
+  },
+  {
+    "id": 5,
+    "title": "Wild Seed",
+    "author": "Octavia E. Butler",
+    "genres": ["fantasy"],
+    "price": 5.00
+  },
+  …
+]
+```
+
+If you are using this dataset in a webshop, you might want to allow your users to sort on `author` and `price`:
+
+<CodeSamples id="sorting_guide_update_sortable_attributes_1" />
+
+### Customize ranking rule order (optional)
+
+When users sort results at search time, [MeiliSearch's ranking rules](/learn/core_concepts/relevancy.md) are set up so the top matches emphasize relevant results over sorting order. You might need to alter this behavior depending on your application's needs.
+
+This is the default configuration of MeiliSearch's ranking rules:
+
+```json
+[
+  "words",
+  "typo",
+  "proximity",
+  "attribute",
+  "sort",
+  "exactness"
+]
+```
+
+`"sort"` is in fifth place. This means it acts as a tie-breaker rule: MeiliSearch will first place results closely matching search terms at the top of the returned documents list and only then will apply the `"sort"` parameters as requested by the user. In other words, by default MeiliSearch provides a very relevant sorting.
+
+Placing `"sort"` ranking rule higher in the list will emphasize exhaustive sorting over relevant sorting: your results will more closely follow the sorting order your user chose, but will not be as relevant.
+
+#### Example
+
+If your users care more about finding cheaper books than they care about finding specific matches to their queries, you can place `sort` much higher in the ranking rules:
+
+<CodeSamples id="sorting_guide_update_ranking_rules_1" />
+
+## Sorting results at search time
+
+After configuring `sortableAttributes`, you can use the [`sort` search parameter](/reference/features/search_parameters.md#sort) to control the sorting order of your search results.
+
```

Comment:
Same as comment in searrch parameters. 

It lets the user quickly understand how the string is constructed without having to look at the examples.

```
Usage: `attribute:order`
A search formula is composed of an attribute and an order. Both are separated by a colon `:`. For example, `price:asc`. 
```

What do you think?

Reply:
Yeah, that's great. Will add this.